glue.R

```diff
@@ -0,0 +1,115 @@
+guild_to_parameter <- function(type, 
+                               range = NULL,
+                               inclusive = c(TRUE, TRUE), 
+                               values = NULL) {
+  type <- rlang::arg_match(type, c("double", "integer", "character", "logical"))
```

Comment:
In lieu of a full package structure, can you add some `require()` statements so that we have some idea of the dependencies?

Reply:
I would also be good to preemptively write some roxygen comments for the parameters as well as add some comments for non-tidymodels folks who look at this.

Reply:
resolved in [ead87e3](https://github.com/topepo/guild-ai-r-tidymodels/pull/1/commits/ead87e3cf959f37861c974453f1d26f094e46514) and [befaab6](https://github.com/topepo/guild-ai-r-tidymodels/pull/1/commits/befaab6e049c75cf158856ebf77bc7225b645655)